source/runtime.cpp

```diff
@@ -1265,9 +1306,9 @@ namespace reshade
 		{
 			ImGui::PushStyleVar(ImGuiStyleVar_ItemSpacing, ImGui::GetStyle().ItemSpacing * 2);
 
-			const char *const menu_items[] = { "Home", "Settings", "Statistics", "About" };
+			const char *const menu_items[] = { "Home", "Settings", "Depth Buffer detection settings", "Statistics", "About" };
```

Comment:
This really shouldn't be another menu entry, but rather a section under "Settings".

Reply:
I found it clearer like that, but OK, will move it to the settings tab ;-)

Reply:
I found it clearer like that, but OK, will move it to the Settings tab ;-)